assets/asm/base/gamemode.asm

```diff
@@ -59,6 +59,11 @@ freecode
 		STA $00			
 		LDA $932A,x		
 		STA $01			
-		STZ $02						
+if !bank8
```

Comment:
Could you change the statement to "if !bank8 != $00", making it more clear what it does?